File: data simulator/models/product.py
class Product:
    def __init__(self):
        self.product_id         = None
        self.description        = None
        self.measurment_value   = None
        self.picture            = None
        self.price              = None
        self.product_name       = None
        self.category_id        = None
        self.measurment_id      = None
        self.seller_id          = None

    def __str__(self):
        return f"""
    {{
        "product_name":"{self.product_name}",
        "category_id": {self.category_id},
        "seller_id": {self.seller_id},
        "measurement_id":{self.measurment_id},
        "picture":"",
        "price":{self.price},
        "description":"{self.description}"
    }}
    """

File: server/src/main/kotlin/server/server/models/Product.java
package server.server.models;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "products")
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "product_id")
    private Long productId;

    @ManyToOne
    @JoinColumn(name="seller_id")
    private Seller seller;


    @Column(name="product_name")
    private String productName;
    private byte[] picture;
    private String description;
    private double price;

    @ManyToOne()
    @JoinColumn(name="category_id")
    private Category category;

    @ManyToOne()
    @JoinColumn(name="measurement_id")
    private Measurement measurement;

//  Opciono
    @Column(name="measurement_value")
    private String measurement_value;

    private boolean available;
}
